integration-tests/features/environment.py

```diff
@@ -200,6 +203,18 @@ def _wait_for_system(context, wait_for_server=60):
                         format(s=wait_for_server))
 
 
+def _wait_for_jobs_debug_api_service(context, wait_for_service=60):
+    start = datetime.datetime.now()
+    wait_till = start + datetime.timedelta(seconds=wait_for_service)
+
+    while datetime.datetime.now() < wait_till:
```

Comment:
@fridex do you mean: for _ in range(wait_for_service)? given the sleep is set to 1, it's nearly the same, thx